backtester/eventhandlers/exchange/exchange.go

```diff
@@ -104,6 +104,26 @@ func (e *Exchange) ExecuteOrder(o order.Event, data data.Handler, bot *engine.En
 		limitReducedAmount = reducedAmount
 	}
 
+	// Conforms the amount to fall into the minimum size and maximum size limit after reduced
+	switch f.GetDirection() {
```

Comment:
This does not also assess the global minimums, however I think this is a good improvement 👍 . I think I'll extract the sizing functions so that both the exchange and portfolio managers can use them in a future PR.